src/frontend/src/components/DateFilterMenu/constants.ts

```diff
@@ -37,3 +37,7 @@ export const MENU_OPTIONS_COLLECTION_DATE: DateMenuOption[] = [
     numDaysStartOffset: 365,
   },
 ];
+
+export const MENU_OPTION_ALL_TIME: DateMenuOption = {
+  name: "All time",
```

Comment:
should this be `All Time`? (haven't looked at the design spec, jw)

Reply:
yep!